malevo/swagger_v2.yml

```diff
@@ -0,0 +1,169 @@
+swagger: "2.0"
+info:
+  description: "This is the initial API proposal for the ConfusionFlow REST API"
+  version: "0.0.1"
+  title: "ConfusionFlow"
+  termsOfService: ""
+  contact:
+    email: "gfrogat@gmail.com"
+host: "localhost:8080"
+basePath: "/v2"
+tags:
+- name: "run"
+  description: ""
+- name: "view"
+  description: ""
+schemes:
+- "https"
+- "http"
+paths:
+  /run:
```

Comment:
Why not call it `runs` since it returns all available runs?

Reply:
missed that, sorry